### PR TITLE
Ore Respawn Part 1

### PR DESCRIPTION
**Pull request by NickTyrer**
## 🍰 Pullrequest
Please do not make any additions or changes to this PR

### CODE REVIEW

File: sql/migrations/20220501213609_world.sql [hunk 1]
```diff
@@ -0,0 +1,4654 @@
...
+UPDATE `gameobject` SET `position_x`=-7126.65, `position_y`=-2853.83, `position_z`=15.9532, `orientation`=3.05433, `rotation0`=0, `rotation1`=0, `rotation2`=0.999048, `rotation3`=0.0436193 WHERE `guid`=7156;
+
+-- Correct position of Mithril Deposit in Searing Gorge (position is off by 1.82687 yards).
+UPDATE `gameobject` SET `position_x`=-7076.41, `position_y`=-1297.19, `position_z`=267.191, `orientation`=5.98648, `rotation0`=0, `rotation1`=0, `rotation2`=-0.147809, `rotation3`=0.989016 WHERE `guid`=7300;
+
+-- Correct position of Mithril Deposit in Searing Gorge (position is off by 0.0943908 yards).
+UPDATE `gameobject` SET `position_x`=-6862.05, `position_y`=-1551.08, `position_z`=242.068, `orientation`=3.10665, `rotation0`=0, `rotation1`=0, `rotation2`=0.999847, `rotation3`=0.0174693 WHERE `guid`=7079;
+
+-- Correct position of Mithril Deposit in Western Plaguelands (position is off by 1.60459 yards).
+UPDATE `gameobject` SET `position_x`=1082.49, `position_y`=-1079.31, `position_z`=100.575, `orientation`=2.80998, `rotation0`=0, `rotation1`=0, `rotation2`=0.986285, `rotation3`=0.16505 WHERE `guid
```
Comment (AnonXS) [inline sql/migrations/20220501213609_world.sql]: dont think this is intended.

File: sql/migrations/20220501213609_world.sql [hunk 2]
```diff
@@ -5012,9 +5012,27 @@ UPDATE `pool_gameobject` SET `pool_entry` = 1274 WHERE `guid` IN (397176, 397177
...
--- SELECT * FROM gameobject WHERE id IN (SELECT entry FROM gameobject_template WHERE NAME LIKE "%Vein%") AND rotation2 = 0
--- SELECT * FROM gameobject WHERE id IN (SELECT entry FROM gameobject_template WHERE NAME LIKE "%Deposit%") AND rotation2 = 0
+/*
+LOOK AT ALL THE CUSTOM ORE OBJECTS
+SELECT * FROM gameobject WHERE id IN (SELECT entry FROM gameobject_template WHERE NAME LIKE "%Vein%") AND rotation2 = 0
+SELECT * FROM gameobject WHERE id IN (SELECT entry FROM gameobject_template WHERE NAME LIKE "%Deposit%") AND rotation2 = 0
+*/
+
+/*
+NOTE: CMANGOS HAS LOTS MORE ORES HERE (COMMITED DIRECTLY TO CLASSIC SO NOT BACK-PORTED)
```
Comment (AnonXS) [inline sql/migrations/20220501213609_world.sql]: +1

Comment (ratkosrb) [reply sql/migrations/20220501213609_world.sql]: Do you know what's the source of the data?

Comment (AnonXS) [reply sql/migrations/20220501213609_world.sql]: Looking at our recent classic zone reworks, i can say that alot of spawns are faulty and had to be readjusted.
and thats what i expect of anyone committing to mangos, have brains and just dont go half way of what you're looking at peronally.
I hope people really try to do their best to make the content the best.

**Pull request closed without merging**